### ISSUE TITLE
tab titles, in Task View, doesnot update in real time

### ISSUE THREAD

**Issue by shalva97** [minbrowser/min#1353]
* **Min Version:** 1.16.0
* **Operating system:** Arch

## Expected Behavior
tabs, in Task View, should reflect new title as soon as it is changed

## Actual Behavior
it doesnot update until Task View is reopened

## To Reproduce
register on toggl.com then press "play" button on the top right to start a timer. Tab title should start updating every second to reflect time spent. But if you open Task View, title stops updating

**Comment by PalmerAL** [comment]
This is the same root cause as #357 - the task overlay elements are created when the overlay is opened, and don't update in response to any events.

Is there a scenario where you need the timer to update while the overlay is open, or is this just something you noticed?

**Comment by shalva97** [comment]
yeah, it is not that important. But real time updates would be cool...
